<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgColumnDef</class>
 <widget class="QDialog" name="DlgColumnDef">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Column definition</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2_2_2_2_2">
         <property name="text">
          <string>Default &amp;value</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="buddy">
          <cstring>edDefaultValue</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="edDecimals"/>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="chkPrimaryKey">
         <property name="text">
          <string>&amp;Primary key</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edLength"/>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="chkNotNull">
         <property name="text">
          <string>NOT NU&amp;LL</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="chkUnsigned">
         <property name="text">
          <string>UNSIGNED</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2_3">
         <property name="text">
          <string>&amp;Type</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="buddy">
          <cstring>lstType</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Name</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="buddy">
          <cstring>edName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2_2_3">
         <property name="text">
          <string>&amp;Length</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="buddy">
          <cstring>edLength</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2_2_2_3">
         <property name="text">
          <string>&amp;Decimals</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="buddy">
          <cstring>edDecimals</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>References</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="chkUnique">
         <property name="text">
          <string>&amp;UNIQUE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edName"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="edDefaultValue"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="lstType"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Character set</string>
         </property>
         <property name="buddy">
          <cstring>lstCharacterSet</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="lstCharacterSet">
         <item>
          <property name="text">
           <string>default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Utf8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Latin1</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="lstRefTable">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="lstRefColumn">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Collation</string>
         </property>
         <property name="buddy">
          <cstring>lstCharacterSet</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="lstCollation"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="grpEnum">
       <property name="title">
        <string>Enum/Set</string>
       </property>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QTextEdit" name="txtEnum">
          <property name="toolTip">
           <string>one entry per line</string>
          </property>
          <property name="styleSheet">
           <string>color: black</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Nimbus Sans L'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Comment</string>
       </property>
       <property name="buddy">
        <cstring>txtComment</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtComment">
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>407</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="chkShowCommand">
       <property name="toolTip">
        <string>Preview alter command before execution. Execution can be canceled then.</string>
       </property>
       <property name="text">
        <string>Preview command</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edName</tabstop>
  <tabstop>lstType</tabstop>
  <tabstop>edLength</tabstop>
  <tabstop>edDecimals</tabstop>
  <tabstop>edDefaultValue</tabstop>
  <tabstop>chkNotNull</tabstop>
  <tabstop>chkUnique</tabstop>
  <tabstop>btOk</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DlgColumnDef</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btOk</sender>
   <signal>clicked()</signal>
   <receiver>DlgColumnDef</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
